test/support/auth_helper_case.ex

```diff
@@ -0,0 +1,50 @@
+defmodule ElixirGoogleScraperWeb.AuthHelperCase do
+  use ExUnit.CaseTemplate
```

Comment:
```suggestion
defmodule ElixirGoogleScraperWeb.AuthHelper do
```
I think we can use a simple module here. It just a collection of helper functions. 👍